java/avro/samples/kafka-consumer/pom.xml

```diff
@@ -23,7 +23,7 @@
     <dependency>
       <groupId>com.microsoft.azure</groupId>
       <artifactId>azure-schemaregistry-kafka-avro</artifactId>
-      <version>1.1.0-beta.1</version>
+      <version>1.1.0-beta.2</version>
```

Comment:
As part of the change, let's add the SpringBoot sample as well. basically have one extra sample that based on the current simple produce/fetch but through https://learn.microsoft.com/en-us/azure/developer/java/spring-framework/authentication way of auth.